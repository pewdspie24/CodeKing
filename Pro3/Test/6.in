100
9 97031442
240690 303089
291173 630534
753126 784932
361590 916626
539539 269647
836572 312597
990999 585943
409187 379498
812759 975325
