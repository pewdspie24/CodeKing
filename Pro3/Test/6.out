185
183
181
179
177
175
173
172
170
168
166
164
162
160
158
156
155
153
151
149
147
145
143
141
139
138
136
134
132
130
128
126
124
122
120
119
117
115
113
111
109
107
105
103
102
100
98
96
94
92
90
88
86
85
83
81
79
77
75
73
71
69
68
66
64
62
60
58
56
54
52
51
49
47
45
43
41
39
37
35
33
32
30
28
26
24
22
20
18
16
15
13
11
9
7
5
3
1
1
1
